desktop-widgets/divelistview.cpp

```diff
@@ -679,30 +679,15 @@ void DiveListView::mergeTripBelow()
 
 void DiveListView::removeFromTrip()
 {
-	//TODO: move this to C-code.
-	int i;
-	struct dive *d;
-	QVector<dive *> divesToRemove;
-	for_each_dive (i, d) {
-		if (d->selected && d->divetrip)
```

Comment:
Attention: the `d->divetrip` condition is lost and must now be moved to the undo-command.

Reply:
So, I don't think you implemented that on the undo-command side, or did I miss that in my review?
Which means we shouldn't merge, yet, correct?

Reply:
Done.